test/setup/node.js

```diff
@@ -0,0 +1,60 @@
+var chai = require('chai');
+var sinon = require('sinon');
+var sinonChai = require('sinon-chai');
+var chaiJq = require('chai-jq');
+var requireHelper = require('./require-helper');
+
+chai.use(sinonChai);
+chai.use(chaiJq);
+
+global.expect = chai.expect;
+global.sinon = sinon;
+
+if (!global.document || !global.window) {
+  var jsdom = require('jsdom').jsdom;
+
+  global.document = jsdom('<html><head><script></script></head><body></body></html>', null, {
+    FetchExternalResources:   ['script'],
+    ProcessExternalResources: ['script'],
+    MutationEvents:           '2.0',
+    QuerySelector:            false
+  });
+
+  global.window = document.parentWindow;
+  global.navigator = global.window.navigator;
+
+  global.window.Node.prototype.contains = function(node) {
+    return this.compareDocumentPosition(node) && 16;
+  };
+}
+
+global.$ = global.jQuery = require('jquery');
+global._ = require('underscore');
+global.Backbone = require('backbone');
+global.Backbone.$ = global.$;
+global.Metal = require('backbone-metal');
+global.Marionette = Backbone.Marionette = {};
+require('backbone.babysitter');
+require('backbone.radio');
+global.slice = Array.prototype.slice;
+// requireHelper('bind-entity-events');
```

Comment:
So something the boilerplate does quite well is keep you from having to maintain what to require in..  I _think_ we could lose the `requireHelper` stuff and just setup jsDOM and Chai.. and then `require('babel/register');  require('./setup')();`  Then in the setup file, require in the dependencies, and just the main marionette src file.